From fe504175b53780c0ebbbef009ff29f0ed9c4592f Mon Sep 17 00:00:00 2001
From: ktietz <ktietz@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Tue, 10 Feb 2015 14:14:58 +0000
Subject: gcov-tool.c: Remove wrong #if !defined(_WIN32)

2015-02-10  Rainer Emrich  <rainer@emrich-ebersheim.de>

PR gcov-profile/61889
* gcov-tool.c: Remove wrong #if !defined(_WIN32)

Part of SVN 220584
---
 gcc/gcov-tool.c | 4 ----

diff --git a/gcc/gcov-tool.c b/gcc/gcov-tool.c
index d63ad1c102e..fd27d7c11ad 100644
--- a/gcc/gcov-tool.c
+++ b/gcc/gcov-tool.c
@@ -99,11 +99,7 @@ gcov_output_files (const char *out, struct gcov_info *profile)
   /* Try to make directory if it doesn't already exist.  */
   if (access (out, F_OK) == -1)
     {
-#if !defined(_WIN32)
       if (mkdir (out, S_IRWXU | S_IRWXG | S_IRWXO) == -1 && errno != EEXIST)
-#else
-      if (mkdir (out) == -1 && errno != EEXIST)
-#endif
         fatal_error (input_location, "Cannot make directory %s", out);
     } else
       unlink_profile_dir (out);
-- 
